content/practices/event-storming-process-modelling/index.md

```diff
@@ -0,0 +1,160 @@
+     - Add a grammar guide as shown  ![This is the placeholder for grammar](/images/practices/event-storming-process-modelling/Grammar.PNG)
+
+1. First Narration: Attempt to narrate a `golden path` through the board, describing the process of events as they happen from left to right. Adjust and add grammar as necessary to reach a consensus of the process.
+    - Roles: 
+       - There are typically 3 roles; one Narrater, one Scribe and the remaining participants are The Audience
+         - A Narrator reads through the process from left to right
+         - The Audience listens to the narrator and assists in discovering the process
+         - A Scribe records the process on the board
+       - Any participant can take on any of these roles and it should be encouraged for roles to rotate
+
+    - Questions: If a question cannot be answered straight away then leave it on the board and return to it later.
+    - Grammar: Add grammar appropriately but do not get bogged down by it
+      1. Events: These represent events that are "Interesting" to the wider system. When understanding what "interesting" means, you might refer to the description given by Martin Fowler in his blog about Event-Driven systems
+      2. Commands: An action that must be performed due to policy, and produces one or more outcomes
+      3. Policies: This sticky represents a decision to do something: "When `<previous event>` `<actor>` `<always, sometimes, if.. then>` `<task>`
+      4. Actors: This sticky represents a human interaction and should be placed with policies to identify who is responsible of making the decision and performing the task, a policy without an actor is automated
+      5. Tools: external/3rd party tools such as; email, excel, notepad, or any required COTS application
+      6. Read Models: Data required to make decisions or perform commands
+      Note: it does not always need to happen in this order and more grammar may be added, so it is best to leave enough space or be prepared to shift post-its further to the right as you make changes.
+    - Negative/Alternative Outcomes: If a command can result in multiple or alternative events then all events should be placed vertically
+    - Pain Points and Opportunities: Place a sticky to represent any pain points and any opportunities that are identified at the time. Pain points can be around the process itself or the current system or tools supporting the process
+
+3. Narrate the process again, including the policies and commands and add or correct any grammar along the way. 
+   You can repeat this until you have agreed on at least one end-to-end slice through the process
+
+4. Review Questions and Pain Points: Try to answer as many questions as possible and rationalise pain points and opportunities
```

Comment:
Please use markdown syntax for numbered lists. See above.

Reply:
fixed